utils/azure/identity/default_credentials.cc

```diff
@@ -75,15 +75,15 @@ future<default_credentials::credentials_opt> default_credentials::get_credential
     auto creds_found = tenant_id || client_id || client_secret || client_certificate_path;
     auto creds_complete = tenant_id && client_id && (client_secret || client_certificate_path);
     if (!creds_found) {
-        az_creds_logger.debug("[{}] No credentials found in environment", *this);
+        az_creds_logger.info("[{}] No credentials found in environment", *this);
         co_return std::nullopt;
     } else if (!creds_complete) {
-        az_creds_logger.debug("[{}] Incomplete credentials. Both 'AZURE_TENANT_ID' and 'AZURE_CLIENT_ID', and at least one of 'AZURE_CLIENT_SECRET', 'AZURE_CLIENT_CERTIFICATE_PATH' must be provided. Currently:", *this);
-        az_creds_logger.debug("[{}] Tenant ID is {}set", *this, tenant_id ? "" : "NOT ");
-        az_creds_logger.debug("[{}] Client ID is {}set", *this, client_id ? "" : "NOT ");
-        az_creds_logger.debug("[{}] Client Secret is {}set", *this, client_secret ? "" : "NOT ");
-        az_creds_logger.debug("[{}] Client Certificate is {}set", *this, client_certificate_path ? "" : "NOT ");
-        az_creds_logger.debug("[{}] Authority host is {}set", *this, authority ? "" : "NOT ");
+        az_creds_logger.info("[{}] Incomplete credentials. Both 'AZURE_TENANT_ID' and 'AZURE_CLIENT_ID', and at least one of 'AZURE_CLIENT_SECRET', 'AZURE_CLIENT_CERTIFICATE_PATH' must be provided. Currently:", *this);
+        az_creds_logger.info("[{}] Tenant ID is {}set", *this, tenant_id ? "" : "NOT ");
+        az_creds_logger.info("[{}] Client ID is {}set", *this, client_id ? "" : "NOT ");
+        az_creds_logger.info("[{}] Client Secret is {}set", *this, client_secret ? "" : "NOT ");
+        az_creds_logger.info("[{}] Client Certificate is {}set", *this, client_certificate_path ? "" : "NOT ");
+        az_creds_logger.info("[{}] Authority host is {}set", *this, authority ? "" : "NOT ");
```

Comment:
This will interfere with other logs and will intermix different lines with unrelated (to azure creds) logging. If touching this place, how about merging them into a single info() invocation?